### ISSUE TITLE
Skipped test: ValidateCSharpTemplateProjects blazor and blazorwasm

### ISSUE THREAD

**Issue by 333fred** [dotnet/roslyn#80263]
```
  Failed Microsoft.CodeAnalysis.MSBuild.UnitTests.NewlyCreatedProjectsFromDotNetNew.ValidateCSharpTemplateProjects(templateName: "blazor") [5 s]
  Error Message:
   Expected 0 items but found 2: The following compiler diagnostics are being reported for the template.
Items:
    C:\h\w\AC3208FD\t\RoslynTests\a7627a86-1c6e-45d2-93e3-9edd1d6493ab\Program.cs(1,7): error CS0246: The type or namespace name 'a7627a86_1c6e_45d2_93e3_9edd1d6493ab' could not be found (are you missing a using directive or an assembly reference?)
    C:\h\w\AC3208FD\t\RoslynTests\a7627a86-1c6e-45d2-93e3-9edd1d6493ab\Program.cs(24,24): error CS0246: The type or namespace name 'App' could not be found (are you missing a using directive or an assembly reference?)
  Stack Trace:
     at Roslyn.Test.Utilities.AssertEx.Empty[T](IEnumerable`1 items, String message) in /_/src/Compilers/Test/Core/Assert/AssertEx.cs:line 873
   at Microsoft.CodeAnalysis.MSBuild.UnitTests.NewlyCreatedProjectsFromDotNetNew.<<AssertTemplateProjectLoadsCleanlyAsync>g__AssertProjectLoadsCleanlyAsync|9_3>d.MoveNext() in /_/src/Workspaces/MSBuild/Test/NewlyCreatedProjectsFromDotNetNew.cs:line 239
```

```
  Failed Microsoft.CodeAnalysis.MSBuild.UnitTests.NewlyCreatedProjectsFromDotNetNew.ValidateCSharpTemplateProjects(templateName: "blazorwasm") [16 s]
  Error Message:
   Expected 0 items but found 2: The following compiler diagnostics are being reported for the template.
Items:
    C:\h\w\AC3208FD\t\RoslynTests\9b8d579d-0509-4f70-a2b7-53d9f9a13f05\Program.cs(3,7): error CS0246: The type or namespace name '_9b8d579d_0509_4f70_a2b7_53d9f9a13f05' could not be found (are you missing a using directive or an assembly reference?)
    C:\h\w\AC3208FD\t\RoslynTests\9b8d579d-0509-4f70-a2b7-53d9f9a13f05\Program.cs(6,28): error CS0246: The type or namespace name 'App' could not be found (are you missing a using directive or an assembly reference?)
  Stack Trace:
     at Roslyn.Test.Utilities.AssertEx.Empty[T](IEnumerable`1 items, String message) in /_/src/Compilers/Test/Core/Assert/AssertEx.cs:line 873
   at Microsoft.CodeAnalysis.MSBuild.UnitTests.NewlyCreatedProjectsFromDotNetNew.<<AssertTemplateProjectLoadsCleanlyAsync>g__AssertProjectLoadsCleanlyAsync|9_3>d.MoveNext() in /_/src/Workspaces/MSBuild/Test/NewlyCreatedProjectsFromDotNetNew.cs:line 239
```

**Comment by MihuBot** [comment]
I'm a bot. Here is a possible related and/or duplicate issue (I may be wrong):
- https://github.com/dotnet/roslyn/issues/72015

**Comment by jasonmalinowski** [comment]
Created https://github.com/dotnet/roslyn/pull/80468 to see if these fail in CI. Trying them locally, they still pass.
